src/components/main.js

```diff
@@ -91,24 +92,24 @@ class Main extends React.Component {
       window.onload = this.handleKeyringOpener();
       this.setState({ name: keyringName }, () => {
         // Check if this keyring has already logged in. This login should expire after a period of time.
-        const prevKeyringLogin = this.getPrevKeyringLogin();
+        const prevKeyringLogin = localStorage.getKeyringItem(keyringName);
         const keyringTimeoutBoundary = new Date().getTime() - constants.KEYRING_LOGOUT_MS;
         if (!forceLogin && prevKeyringLogin && prevKeyringLogin.lastLogin > keyringTimeoutBoundary) {
           this.connect( prevKeyringLogin.deviceID, 
                         prevKeyringLogin.password, 
                         () => this.connectSession(prevKeyringLogin));
         } else {
           // If the login has expired, clear it now.
-          this.clearPrevKeyringLogin();
+          localStorage.removeKeyringItem(keyringName)
         }
       })
     } else if (hwCheck) {
       // Lattice validation check builds this URL and includes a signature + preimage
       this.setState({ hwCheck })
     } else {
       // Lookup deviceID and pw from storage
-      const deviceID = window.localStorage.getItem('gridplus_web_wallet_id');
-      const password = window.localStorage.getItem('gridplus_web_wallet_password');
+      const deviceID = localStorage.getWalletId()
+      const password = localStorage.getWalletPassword()
```

Comment:
These should be renamed to `getLoginId` and `getLoginPassword`. This app used to be called the "web wallet" but wallet functionality is much less emphasized now